libraries/AP_Airspeed/AP_Airspeed_DroneCAN.cpp

```diff
@@ -13,21 +13,13 @@ extern const AP_HAL::HAL& hal;
 AP_Airspeed_DroneCAN::DetectedModules AP_Airspeed_DroneCAN::_detected_modules[];
 HAL_Semaphore AP_Airspeed_DroneCAN::_sem_registry;
 
-void AP_Airspeed_DroneCAN::subscribe_msgs(AP_DroneCAN* ap_dronecan)
+bool AP_Airspeed_DroneCAN::subscribe_msgs(AP_DroneCAN* ap_dronecan, uint8_t driver_index)
 {
-    if (ap_dronecan == nullptr) {
```

Comment:
i don't think the removal of the driver_index helps us move towards having the subscribes happen directed by the library subsystems instead of by AP_DroneCAN

Reply:
having: 
```
const auto driver_index = ap_dronecan->get_driver_index();
```
may make things more readable while keeping the interface simple